manifests/0000_70_cluster-network-operator_01_egr_crd.yaml

```diff
@@ -0,0 +1,151 @@
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: EgressRouterSpec contains the configuration for an egress router.
+            properties:
+              ip:
+                description: IP configuration arguments.
+                properties:
+                  addresses:
+                    description: List of IP addresses to configure on the interface.
+                    items:
+                      description: EgressRouterAddress contains a pair of ip cidr and gateway to be configured on the router's interface
+                      properties:
+                        gateway:
+                          description: IP address of the next-hop gateway, if it cannot be automatically determined. Can be IPv4 or IPv6.
+                          type: string
+                        ip:
+                          description: IP is the address to configure on the router's interface. Can be IPv4 or IPv6.
+                          type: string
+                      required:
+                      - ip
+                      type: object
+                    type: array
+                  allowedDestinations:
+                    description: List of CIDR blocks that the pod is allowed to connect to via this interface. If not provided, the pod can connect to any destination.
+                    items:
+                      description: CIDR represents an IPv4 or IPV6 CIDR block.
+                      type: string
+                    type: array
+                required:
+                - addresses
+                type: object
+              networkInterface:
+                description: Specification of interface to create/use. e.g; macvlan
+                properties:
+                  macVLAN:
+                    description: Arguments specific to the interfaceType macvlan
+                    properties:
+                      master:
+                        description: Name of the master inteface. Need not be specified if it can be inferred from the IP address.
+                        minLength: 1
+                        type: string
+                      mode:
+                        description: Mode depicts the mode that is used for the macvlan interface. The default mode is "vepa".
```

Comment:
The default mode is bridge. 
https://github.com/openshift/egress-router-cni/blob/master/pkg/macvlan/macvlan.go#L198-L200

Also I would expect the description to include something like:
```
The following modes are allowed:
- Bridge
- Private
- VEPA (Or Vepa, whatever you decide to do)
- Passthrough
```
The reason why I expect this is because `oc explain` (or `kubectl`) don't show the enum values, you need to document them manually.

Reply:
hmm yea actually you are right, will update this.